tests/Application/Actions/Donations/UpdateHandlesLockExceptionTest.php

```diff
@@ -184,12 +193,11 @@ public function setExpectationsForPersistAfterRetry(string $donationId, Donation
             return null;
```

Comment:
I know this is outside the diff, and probably my mistake originally, but it seems weird that the `$newStatus` param is unused. Maybe we should assert on it like this?

```suggestion
            TestCase::assertEquals($newStatus, $donation->getDonationStatus());
            return null;
```
Need to also `use` $newStatus and $donation at line 188.